server/views/layout.njk

```diff
@@ -36,7 +36,7 @@
       tag: {
         text: "beta"
       },
-      html: 'This is a new service – your <a class="govuk-link" href="#">feedback</a> will help us to improve it.'
+      html: 'This is a new service – your <a class="govuk-link" href="'+ phaseBannerLink +'">feedback</a> will help us to improve it.'
```

Comment:
`rel="noreferrer noopener"` here too please

Reply:
I'll add this too - just a sec.

Reply:
Done now.